bundles/automation/org.eclipse.smarthome.automation.module.script.rulesupport/src/main/java/org/eclipse/smarthome/automation/module/script/rulesupport/internal/LoaderScriptExtension.java

```diff
@@ -81,14 +84,17 @@
         staticTypes.put("TriggerType", TriggerType.class);
         staticTypes.put("ConfigDescriptionParameter", ConfigDescriptionParameter.class);
 
+        staticTypes.put("ItemRegistry", null);
+        staticTypes.put("ir", null);
```

Comment:
This will work fine, but I was wondering why you didn't put the scope registration in the DefaultScriptScopeProvider with the ItemRegistryDelegate registration. It seems like a closely related type but one requires an importPreset call and the other doesn't.